service/get_test.go

```diff
@@ -133,6 +135,19 @@ func startHTTPServer() *http.Server {
 	return srv
 }
 
+func waitForServer(url string) {
+	var retry = 10
+	for i := 0; i < retry; i++ {
+		resp, err := http.Get(url)
+		if err != nil {
+			continue
+		}
+		if resp.StatusCode == http.StatusOK {
+			return
+		}
+	}
+}
+
```

Comment:
This is duplicated code, we can move this `waitForServer` into something like `common_test.go` and include it in both the files using it.